casper/src/test/scala/coop/rchain/casper/helper/RhoSpec.scala

```diff
@@ -57,6 +58,7 @@ object RhoSpec {
   def getResults(
       testObject: CompiledRholangSource,
       otherLibs: Seq[DeployData],
+      deployerPk: Option[PublicKey],
```

Comment:
No need for this paramter, it's in testObject

Reply:
Done